tests/assets/layered_deployment.template_2.json

```diff
@@ -0,0 +1,61 @@
+{
+    "content": {
+        "modulesContent": {
+            "$edgeAgent": {
+                "properties.desired": {
+                    "modules": {
+                        "LayeredDeploymentTempSensor": {
+                            "settings": {
+                            "image": "mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:${LAYERED_RUNTIME_TAG}",
+                                "createOptions": ""
+                            },
+                            "type": "docker",
+                            "status": "running",
+                            "restartPolicy": "always",
+                            "version": "1.0"
+                        },
+                        "LayeredDeploymentCSharpModule": {
+                            "version": "1.0",
+                            "type": "docker",
+                            "status": "running",
+                            "restartPolicy": "always",
+                            "settings": {
+                                "image": "${MODULES.csharpmodule.amd64}",
+                                "createOptions": {}
+                            }
+                        },
+                        "LayeredDeploymentCSharpModule2": {
+                            "version": "1.0",
+                            "type": "docker",
+                            "status": "running",
+                            "restartPolicy": "always",
+                            "settings": {
+                                "image": "${MODULES.csharpmodule2}",
```

Comment:
should this be 'MODULES.csharpmodule2.amd64'?

Reply:
I'm referring to deployment.template_3.json in the same directory, so I'm keeping it consistent with that until I start running the tests and tweak things around.